paasta_tools/kubernetes_tools.py

```diff
@@ -3526,3 +3531,60 @@ def get_kubernetes_secret_env_variables(
                 kube_client, secret_name, service_name
             )
     return decrypted_secrets
+
+
+def get_kubernetes_secret_volumes(
+    kube_client: KubeClient,
+    secret_volumes_config: Dict[str, Any],
```

Comment:
if i'm not mistaken, this is could be more tightly typed as `paasta_tools.utils.SecretVolume`

(same in `decrypt_secret_volumes()` and `decrypt_secret()` if this does need to be updated)

Reply:
mmm maybe we're both mistaken 😅 

```
(Pdb) instance_config.get_secret_volumes()
[{'container_path': '/who/config/', 'secret_name': 'config_yaml_file', 'default_mode': '0444', 'items': [{'key': 'config_yaml_file', 'path': 'config.yaml'}]}]
(Pdb) type(instance_config.get_secret_volumes())
<class 'list'>
```

Reply:
Fixed this accordingly to `List[Dict[str, Any]]`

Reply:
`SecretVolume` is a TypedDict (which isn't really a real type - calling `type()` on it will show `dict`) and it looks like `get_secret_volumes()` is defined at https://sourcegraph.yelpcorp.com/Yelp/paasta/-/blob/paasta_tools/utils.py?L911 and is typed as returning List[SecretVolume]

Reply:
(so this should be typed as `List[SecretVolume]` to match `get_secret_volumes()` - we generally wanna avoid using `Any` and using `TypedDict` on top of that makes mypy more powerful + gives IDEs/language servers more info)

Reply:
Fixed by making this `Sequence[SecretVolume]`